### PR TITLE
changed module shoes to class Shoes :issue #450 solution

### PR DESCRIPTION
**Pull request by BLSowmya**
### Description
I have changed module Shoes to class shoes 

I have changed all module Shoes  to class shoes as mentioned in #450

### Checklist

- [x] Run tests locally
- [x] Run linter(check for linter errors)

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```
Comment (gintama91) [inline .tool-versions:1]: we dont keep track using .tool-versions so delete this ig or add this to gitignore file

Comment (BLSowmya) [reply .tool-versions:1]: okay

**Comment by gintama91** [comment]
also it's better practice to checkout to some other branch instead of using main

File: examples/radio/radio_groups.rb [hunk 2]
```diff
@@ -13,6 +13,6 @@
       radio :films
       para "Children of Heaven by Majid Majidi"
     end
-    puts radio(:films).checked?
+    para "Is radio button checked? #{radio(:films).checked?}"
```
Comment (noahgibbs) [inline examples/radio/radio_groups.rb:16]: Nice!

File: lacci/lib/shoes/builtins.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 # Shoes has a number of built-in methods that are intended to be available everywhere,
 # in every Shoes and non-Shoes class, for every Shoes app.
-module Shoes::Builtins
+class Shoes::Builtins
```
Comment (noahgibbs) [inline lacci/lib/shoes/builtins.rb:5]: For this, probably want to keep Builtins as a module.

**Comment by noahgibbs** [comment]
Looks good! We should remove the PR message saying you need to run Rubocop -- you're welcome to, but it's no longer a build requirement.

I'd like you to change the Builtins thing. I should figure out why we don't have a test that would catch that. But other than that, this looks good.

**Comment by noahgibbs** [comment]
Oh yeah, we totally had a test that caught that! Good. I was hoping so.

File: lacci/lib/shoes/builtins.rb [hunk 4]
```diff
@@ -2,59 +2,61 @@
...
-class Shoes::Builtins
+module Shoes::Builtins
   # Register the given font with Shoes so that text that wants it can use it.
   # Also add it to the FONTS constant.
   #
   # @param font_file_or_url [String] the filename or URL for the font
   # @return [void]
-  def font(font_file_or_url)
-    shoes_builtin("font", font_file_or_url)
+  class Shoes::Builtins
```
Comment (gintama91) [inline lacci/lib/shoes/builtins.rb:11]: i think u r supposed to change class back to module not adding module on top of class

Comment (BLSowmya) [reply lacci/lib/shoes/builtins.rb]: yeah ,I have changed it

**Comment by gintama91** [comment]
> hey @BLSowmya can u also do this?
> 
> > Looks good! We should remove the PR message saying you need to run Rubocop -- you're welcome to, but it's no longer a build requirement.
> 
> just change things in pull request template in workflows.

nvm this ig noah comitted this. didn't see earlier

**Review by gintama91 — APPROVED** [review]

**Comment by noahgibbs** [comment]
> nvm this ig noah comitted this. didn't see earlier

Yeah, I did. Sorry, should have mentioned.

**Comment by noahgibbs** [comment]
Looks great, merged!

**Comment by BLSowmya** [comment]
> Looks great, merged!

Thank you!!!

**Comment by noahgibbs** [comment]
And looks like we don't have any Shoes-Spec tests yet that conflict with Shoes being a class instead of a module -- the Shoes-Spec tests all pass fine.

**Review by Schwad — COMMENTED** [review]
Congratulations on your first contribution! 🎉 wooohooo!!!

**Comment by BLSowmya** [comment]
> Congratulations on your first contribution! 🎉 wooohooo!!!

Thank you soo muchh!!

**Pull request merged**